app/forms/admin/participants/change_induction_start_date_form.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Admin::Participants
+  class ChangeInductionStartDateForm
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+    include ::ActiveRecord::AttributeAssignment
+
+    attribute :induction_start_date, :date
+
+    validates :induction_start_date, presence: true
+    # TODO: other validation?
```

Comment:
Not sure what/if there's anything we should be checking here, especially as it's on the admin side.

Reply:
I see some specs in the participant_validation_service_spec.rb related to the `induction_start_date`. Any chances we need to run the ParticipantValidationService when changing the induction_start_date?

Reply:
Yeah - I saw that too. From what I could see it looks unrelated, the [`induction_start_date`](https://github.com/DFE-Digital/early-careers-framework/blob/develop/spec/services/participant_validation_service_spec.rb#L25) variable just clashes with the name of the attribute, but actually represents an `InductionRecord#start_date`.

Looking through the code I can't even see where `#induction_start_date is used, other than displaying it in the admin panel. It was only added in July (407a705bd2).

Also, having just looked at the ticket comments, Mili said:

> confirmed that there are no contractual constraints on induction start date so we don’t need to apply any validation/constraints on changing this

Looks like this should be ok, providing it's targetting the right date. Should it really be changing the start date of the latest induction record? @tonyheadford might have a better idea.

Reply:
Yes thats fine as long as there is understanding that, at the moment at least, the `start_date` on the first induction record isn't necessarily the same as the `induction_start_date`.  There's already plenty of scope for misunderstanding :)